pandas/tests/groupby/test_reductions.py

```diff
@@ -283,8 +283,6 @@ def test_idxmin_idxmax_extremes_skipna(skipna, how, float_numpy_dtype):
                     np.nan,
                     max_value,
                     np.nan,
-                    np.nan,
-                    np.nan,
```

Comment:
this change is bc there's an all-NA group so itll now raise?

Reply:
Yes, and this tested for elsewhere.